docs/src/mlj_cheatsheet.md

```diff
@@ -96,20 +97,26 @@ y, X =  unpack(channing,
 
 Splitting row indices into train/validation/test, with seeded shuffling:
 
-`train, valid, test = partition(eachindex(y), 0.7, 0.2, rng=1234)` for 70:20:10 ratio
+```julia-repl
+julia> train, valid, test = partition(eachindex(y), 0.7, 0.2, rng=1234) # for 70:20:10 ratio
+```
 
```

Comment:
I think in a cheatsheet we should just suppress all `julia>`. As far as I remember, all code is input (no output is displayed) for purposes of concision.